lib/parser.js

```diff
@@ -98,9 +100,18 @@ function _mapSymbol(symbol){
     symbol.type = types[0].types.join(' | ');
   }
 
-  if(returns.length === 1){
-    symbol["return"] = returns[0].types.join(' ');
-    symbol["returnDescription"] = returns[0].description.replace(/(<([^>]+)>)/ig, '');
```

Comment:
current jade template:
https://github.com/scottnath/pattern-library-utilities/blob/master/lib/templates/doxx.template.jade

My plan is to write something which can parse the @requires section to link out to NPM or to local files. Before I do...ever seen anything like that?